### ISSUE TITLE
Is it normal to not applying softmax before F.cross_entropy?

### ISSUE THREAD

**Issue by MilkClouds** [open-mmlab/mmclassification#968]
### Checklist

- I have searched related issues but cannot get the expected help.
- I have read related documents and don't know what to do.

### Describe the question you meet

Hello, I'm using mmcls to classify some images. When I'm using `swin_transformer/swin-base_16xb64_in1k-384px.py`, I noticed that `LinearClsHead` do not apply softmax and pass arguments to loss, which is `CrossEntropyLoss` on `swin-base_16xb64_in1k-384px.py`.

In my knowledge, it is necessary to apply softmax before calculating cross entropy in loss function. But even without this it seems that training is going well, without significant problem. I'm curious about this issue. 

### Post related information

1. The output of `pip list | grep "mmcv\|mmcls\|^torch"`
   \[here\]
2. Your config file if you modified it or created a new one.

I used config file `swin_transformer/swin-base_16xb64_in1k-384px.py` on master branch, without modification.

```python
[here]
```

3. Your train log file if you meet the problem during training.
   \[here\]
4. Other code you modified in the `mmcls` folder.
   \[here\]

**Comment by mzr1996** [comment]
It's right, you can get more details [here](https://pytorch.org/docs/stable/generated/torch.nn.functional.cross_entropy.html#torch.nn.functional.cross_entropy).
